### ISSUE TITLE
What is model.loss?

### ISSUE THREAD

**Issue by xingyifei2016** [iTheresaApocalypse/Quad#1]
When looking at the code at K-FAC.py, I noticed that model.loss and the backward pass is called twice in the function get_ihvp, once in get_ekfac_factors(model, dataloader, device), and another time in get_ihvp itself, over the entire dataset. How is this loss defined? Is this the cross entropy loss? If so, is the model essentially trained on the dataset twice during get_ihvp? 

First call to loss.backward
<img width="484" height="224" alt="Image" src="https://github.com/user-attachments/assets/3d18229a-767c-44ed-87a3-40cc5595bd7a" />

Second call to loss.backward
<img width="463" height="85" alt="Image" src="https://github.com/user-attachments/assets/90b51488-d8a0-491a-9cba-9020645d6eb2" />
